src/components/Tooltip.vue

```diff
@@ -0,0 +1,236 @@
+                </slot>
+            </div>
+            <div class="tooltip-arrow" :class="{ 'arrow-above': position === 'above' }"></div>
+        </div>
+    </teleport>
+</template>
+
+<script>
+export default {
+    name: "Tooltip",
+    props: {
+        /** Whether tooltip is visible */
+        visible: {
+            type: Boolean,
+            default: false
+        },
+        /** Content object to display */
+        content: {
+            type: Object,
+            default: null
+        },
+        /** X position (viewport coordinates) */
+        x: {
+            type: Number,
+            default: 0
+        },
+        /** Y position (viewport coordinates) */
+        y: {
+            type: Number,
+            default: 0
+        },
+        /** Position relative to target element */
+        position: {
+            type: String,
+            default: "below",
+            validator: (value) => [ "above", "below" ].includes(value)
+        }
+    },
+    computed: {
+        tooltipStyle() {
+            return {
+                left: this.x + "px",
+                top: this.y + "px",
+            };
+        },
+
+        statusText() {
+            if (!this.content || this.content === 0) {
+                return this.$t("Unknown");
+            }
+
+            const statusMap = {
+                0: this.$t("Down"),
+                1: this.$t("Up"),
+                2: this.$t("Pending"),
+                3: this.$t("Maintenance")
+            };
+            return statusMap[this.content.status] || this.$t("Unknown");
+        },
+
+        statusClass() {
+            if (!this.content || this.content === 0) {
+                return "status-empty";
+            }
+
+            const classMap = {
+                0: "status-down",
+                1: "status-up",
+                2: "status-pending",
+                3: "status-maintenance"
+            };
+            return classMap[this.content.status] || "status-unknown";
+        },
+
+        timeText() {
+            if (!this.content || this.content === 0) {
+                return "";
+            }
+            return this.$root.datetime(this.content.time);
+        },
+
+        message() {
+            if (!this.content || this.content === 0) {
+                return "";
+            }
+            return this.content.msg || "";
+        }
+    }
+};
+</script>
+
+<style lang="scss" scoped>
+@import "../assets/vars.scss";
+
+.tooltip-wrapper {
+    position: fixed;
+    z-index: 9999;
+    pointer-events: none;
+    transform: translateX(-50%);
+
+    .tooltip-content {
+        background: rgba(17, 24, 39, 0.95);
+        backdrop-filter: blur(8px);
+        border: 1px solid rgba(75, 85, 99, 0.3);
+        border-radius: 8px;
+        padding: 8px 12px;
+        box-shadow: 0 10px 25px rgba(0, 0, 0, 0.25);
+        min-width: 120px;
+        text-align: center;
+
+        .tooltip-status {
+            font-size: 12px;
```

Comment:
I would prefer a little larger text. Otherwise, going down 10px is a little small.

How this looks with 16/13/12:
(we can obviously also go smaller, your PR after all ^^)

![image](https://github.com/user-attachments/assets/31f8f390-9258-4e23-b4f6-00f1ef1870bc)


```suggestion
            font-size: 16px;
```